addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -3028,10 +3028,6 @@ export class Wysiwyg extends Component {
         const payload = ev.data;
         if (evType === 'call_service') {
             this._callService(payload);
-        } else if (evType === 'get_session') {
-            if (payload.callback) {
-                payload.callback(this.env.session);
-            }
```

Comment:
I don't think @Goaman added this just for fun so this is likely breaking something somewhere, probably in website.

Reply:
The event `get_session` was triggered by the function `getSession` from `ServicesMixin` and nothing triggered it without calling this function. `getSession` instead was called but only in file `auth_totp_portal/static/src/js/totp_frontend.js` so I've removed it to read directly on the obj from `@web/session` and so there is no more `get_session` event.